### ISSUE TITLE
[Bug]Texture flickering Zelda TotK 

### ISSUE THREAD

**Issue by AlfreX22** [Ryujinx/Ryujinx#7211]
### Description of the issue

![Ztotkbug2](https://github.com/user-attachments/assets/085ec013-7c2c-4b6f-9b95-eb95fbafb4d3)

looking towards the waterfall, the textures start to flicker

### Reproduction steps

Just by looking at that waterfall in the background you can see the error.

### Log file

[Ryujinx_1.1.1376_2024-08-17_15-59-59.log](https://github.com/user-attachments/files/16646262/Ryujinx_1.1.1376_2024-08-17_15-59-59.log)

### OS

Windows 10

### Ryujinx version

1.1.1376

### Game version

1.1.2

### CPU

Ryzen 5 7600

### GPU

Nvidia RTX 3080

### RAM

32GB

### List of applied mods

none

**Comment by GamerzHell9137** [comment]
Flickering can't be seen on an image. Post a video.

**Comment by richardt94** [comment]
Not OP, but attached video of the same issue. Flickering happens on a variety of water textures (waterfalls, water spheres) and the ultrahand green shadows. My specs:
Ryzen 7 5700X
16 GB RAM
RTX 2080 8 GB
[flicker.webm](https://github.com/user-attachments/assets/e85a44a4-5ff1-4bee-99d2-3340bf837896)

Windows 11

**Comment by AlfreX22** [comment]
yes, it is exactly the same error, thanks for posting the video.

**Comment by GamerzHell9137** [comment]
Yes, its barrier issues, its being worked on.
